sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -147,8 +145,8 @@ Run a model to identify a collection of significant phrases found in the passed-
 ```C# Snippet:ExtractKeyPhrases
 string input = "My cat might need to see a veterinarian.";
 
-ExtractKeyPhrasesResult result = client.ExtractKeyPhrases(input);
-IReadOnlyCollection<string> keyPhrases = result.KeyPhrases;
+Response<IReadOnlyCollection<string>> response = client.ExtractKeyPhrases(input);
+IReadOnlyCollection<string> keyPhrases = response.Value;
```

Comment:
Does this cast to IEnumerable?  If it does, the samples would be cleaner if we used that instead.

Reply:
It does. Could you expand more on why?

Reply:
IReadOnlyCollection is kind of esoteric, and developers will have to think about how to use it, what functionality it provides, etc.  IEnumerable is super common and most .NET developers can use it without thinking hard.  It's fine if it's an IReadOnlyCollection under the hood, but they don't need to know that.